packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -129,11 +129,7 @@ export const mapVisitQueueEntryProperties = (
 });
 
 export function useVisitQueueEntries(currServiceName: string, locationUuid: string): UseVisitQueueEntries {
-  const { queueLocations } = useQueueLocations();
-  const queueLocationUuid = locationUuid ? locationUuid : queueLocations[0]?.id;
-
-  const apiUrl =
-    `${restBaseUrl}/visit-queue-entry?v=full` + (queueLocationUuid ? `&location=${queueLocationUuid}` : '');
+  const apiUrl = `${restBaseUrl}/visit-queue-entry?v=full` + (locationUuid ? `&location=${locationUuid}` : '');
```

Comment:
Just confirm the location passed in the hook is always queue location. Looks okay.